packages/db-ra-data-rest/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@platformatic/db-ra-data-rest",
+  "version": "0.1.0",
+  "description": "Platformatic DB React-Admin REST data provider",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/plaformatic/platformatic.git"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/plaformatic/platformatic/issues"
+  },
+  "homepage": "https://github.com/plaformatic/platformatic#readme",
+  "main": "dist/index.umd.js",
+  "module": "dist/index.es.js",
+  "sideEffects": false,
+  "files": [
+    "*.md",
+    "src"
+  ],
+  "scripts": {
+    "build": "vite build",
```

Comment:
We need linting too. Check: https://github.com/platformatic/platformatic/blob/main/packages/db-dashboard/package.json#L23